sklearn/svm/classes.py

```diff
@@ -44,7 +44,9 @@ class LinearSVC(BaseEstimator, LinearClassifierMixin,
         Tolerance for stopping criteria.
 
     C : float, optional (default=1.0)
-        Penalty parameter C of the error term.
+        Strictly positive parameter that is inversely proportional to the
+        regularization strength and is multiplied with the loss term of the
```

Comment:
```suggestion
        regularization strength and is multiplied with the loss term
```

Reply:
Thanks for the suggestion! And pardon for the delay.